bitnami/ghost/README.md

```diff
@@ -146,6 +146,7 @@ The following table lists the configurable parameters of the Ghost chart and the
 | `smtpPassword`                              | SMTP password                                                                                                         | `nil`                                                        |
 | `smtpFromAddress`                           | SMTP from address                                                                                                     | `nil`                                                        |
 | `smtpService`                               | SMTP service                                                                                                          | `nil`                                                        |
+| `strategy`                                  | The DeploymentStrategy to use when updating (Recreate or RollingUpdate)                                               | `{ type: Recreate }`                                         |
```

Comment:
Is there any reason why we should use "Recreate" as default value?
```console
$ k explain deployment.spec.strategy.type
KIND:     Deployment
VERSION:  apps/v1

FIELD:    type <string>

DESCRIPTION:
     Type of deployment. Can be "Recreate" or "RollingUpdate". Default is
     RollingUpdate.
```
I would use a different approach to ensure users don't try to scale the solution while using "ReadWriteOnce" PVCs. We can create a validation in the `_helpers.tpl` that shows an error if `persistence.accessMode` is not "ReadWriteMany" when `replicaCount` is `>1`

Reply:
There are other alternatives that are less disruptive since they don't imply introducing a breaking change. E.g. using affinity presets.

For instance, we could set by default:
```yaml
podAntiAffinityPreset: ""
podAffinityPreset: hard
persistence:
  enabled: true
  accessMode: ReadWriteOnce
  size: 8Gi
```
By using this, the Ghost deployment would include this section:
```yaml
...
    spec:
      affinity:
        podAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            - labelSelector:
                matchLabels:
                  app.kubernetes.io/name: ghost
                  app.kubernetes.io/instance: ghost
              namespaces:
                - default
              topologyKey: kubernetes.io/hostname
```
Preventing new pods to be scheduled on different K8s nodes so the PVs can be attached to new pods

Reply:
That’s a good idea, I’ll update the PR once I get around to it. Thanks!